kiota_serialization_json/json_serialization_writer.py

```diff
@@ -253,6 +253,26 @@ def write_collection_of_enum_values(
             else:
                 self.value = result
 
+    def write_collection_of_dict_values(
```

Comment:
can we make this method private? we wouldn't want anyone to take a dependency on it.